src/App.js

```diff
@@ -1,25 +1,25 @@
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import { Products } from './pages/Products/Products';
+import { Cart } from './pages/Cart/Cart';
+import { WishList } from './pages/WishList/WishList';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { NavBar } from './components/NavBar/NavBar';
```

Comment:
I have a suggestion to make import statements clean. Add index.js files in each component folder and export the components from there as well, so that you can directly import it from the folder path instead of giving full path.

Reply:
Added the index.js file and importing from the component folder directly